packages/subject-segmentation/src/definitions.ts

```diff
@@ -4,66 +4,102 @@ export interface SubjectSegmentationPlugin {
    *
    * Only available on Android and iOS.
    *
-   * @since 7.0.0
+   * @since 7.2.0
    */
   processImage(options: ProcessImageOptions): Promise<ProcessImageResult>;
+
+
+  /**
+   * Check if the Google Subject Segmentation module is available.
+   *
+   * If the Google Subject Segmentation module is not available, you can install it by using `installGoogleSubjectSegmentationModule()`.
+   *
+   * Only available on Android.
+   *
+   * @since 7.2.0
+   */
+  isGoogleSubjectSegmentationModuleAvailable(): Promise<IsGoogleSubjectSegmentationModuleAvailableResult>;
+  /**
+   * Install the Google Subject Segmentation module.
+   *
+   * **Attention**: This only starts the installation.
+   * The `googleSubjectSegmentationModuleInstallProgress` event listener will
```

Comment:
Please also add the `googleSubjectSegmentationModuleInstallProgress` event listener (see Barcode Scanning plugin).

Reply:
The listener is still missing.